bentoml/service.py

```diff
@@ -553,6 +574,10 @@ def artifacts(self):
     def env(self):
         return self._env
 
+    @property
+    def webui(self):
```

Comment:
`webui` and `_static_files` here seems to be referencing the same thing, maybe rename both to just `web_static_content`? This API technically can also be used to host other static files right?  e.g. it can be a just a markdown documentation page for using this API and allow end-user to download example input json/image files